examples/gallery/embellishments/solar.py

```diff
@@ -15,39 +15,24 @@
 import pygmt
 
 fig = pygmt.Figure()
-# Create a figure showing the global region on a Mollweide projection
-# Land color is set to dark green and water color is set to light blue
-fig.coast(region="d", projection="W0/15c", land="darkgreen", water="lightblue")
-# Set a time for the day-night terminator and twilights, 17:00 UTC on
-# January 1, 2000
+# Create a global map with central longitude 0° East ("d") using a Mollweide ("W")
+# projection with a width of 15 centimeters
+fig.basemap(region="d", projection="W15c", frame=True)
+fig.coast(land="darkgreen", water="lightblue", shorelines="1/1p,gray50")
+
+# Set a time for the day-night terminator and twilights to 17:00 UTC on January 1, 2000
 terminator_datetime = datetime.datetime(
     year=2000, month=1, day=1, hour=17, minute=0, second=0
 )
-# Set the pen line to be 0.5 points thick
-# Set the fill for the night area to be navy blue at different transparency
-# levels
-fig.solar(
-    terminator="day_night",
-    terminator_datetime=terminator_datetime,
-    fill="navyblue@95",
-    pen="0.5p",
-)
-fig.solar(
-    terminator="civil",
-    terminator_datetime=terminator_datetime,
-    fill="navyblue@85",
-    pen="0.5p",
-)
-fig.solar(
-    terminator="nautical",
-    terminator_datetime=terminator_datetime,
-    fill="navyblue@80",
-    pen="0.5p",
-)
-fig.solar(
-    terminator="astronomical",
-    terminator_datetime=terminator_datetime,
-    fill="navyblue@80",
-    pen="0.5p",
-)
+
+# Plot the day-night terminator and twilights
+for terminator in ["day_night", "civil", "nautical", "astronomical"]:
+    fig.solar(
+        terminator=terminator,
+        terminator_datetime=terminator_datetime,
+        # Set the fill for the night area to navy blue with 70 % transparency
```

Comment:
```suggestion
        # Set the fill for the night area to navy blue with 80 % transparency
```

Reply:
Adding in each iteration a filled area with a transparaceny 80 (`fill="navyblue@80"`) looks very blurry and too dark compared to the original version. Not sure if this is a worth improvement.

Reply:
Maybe having a new variable for the transparency level?

Reply:
I am sorry, this is probably coming from the shorelines, I accidentally added (they are removed now).

And I changed the transparency to 85 %, I think now it looks good and quite similar to the old version:

| OLD | NEW 80 % | NEW 85 % | 
| --- | --- | --- |
| ![solar_old](https://github.com/user-attachments/assets/79e56a1d-4d69-4693-9143-1144e0f75c96) | ![solar_new_80](https://github.com/user-attachments/assets/8a4270c2-2dd3-4597-8c6c-d7b69490aa94) | ![solar_new_85](https://github.com/user-attachments/assets/e4247475-9d73-41f5-a4e3-5c0be02bb956) |

Reply:
Yes now it looks better 😉